src/pages/index.jsx

```diff
@@ -1,21 +1,85 @@
 import React from 'react'
-import Button from '../components/Button'
-import Card from '../components/Card'
-import Hero from '../components/Hero'
 import Layout from '../components/Layout'
+import Hero from '../components/Hero'
```

Comment:
There is extra spacing between the Nav and Hero

Reply:
another great catch, thanks!